results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+
+- **JupyterLab does have existing UI customization features by default.** Light versus dark themes, editor font size, and layout modes (Simple Mode, Presentation Mode) all allow users choices to set up JupyterLab in a way that may work better for them. Because these features require users to navigate multiple areas of JupyterLab to get to and do not require any domain-specific knowledge, the test script asked participants to engage with some of these features. The overwhelming response was one of positive surprise. Participants wanted to use these features and did not know they existed. Light/dark themes were the most likely to be known, while font size adjustments weren’t known by any participant. Screen magnifier users and other low-vision participants emphasized how helpful font size adjustment would be for them and that they wished they knew about them before.
+- **JupyterLab’s default—meaning without the addition of extensions—UI customization capabilities are limited.** In conversation with participants and in observing the issues they faced, we’ve determined additional UI customization features that would help them better use JupyterLab. Entirely new customization options include selecting from multiple syntax highlighting themes, being able to select major button colors (like the `--jp-brand-color` family), being able to select state colors (like the warning, error, and info color families), and to set a maximum width for documents in the main document area. Additions to existing customization options include more themes beyond light and dark modes, to be able to change all text sizes in JupyterLab, to be able to select text sizes directly rather than in opaque increments (ie. to set font sizes to a point, pixel, or percent value rather than selecting menu items with no feedback), and to assign other or additional indicators for the status of running or freshly completed code cells.
+
+#### Different options for feedback on cell status 
+
+**Current default JupyterLab feedback for notebook cell status is exclusively visual**; the cell prompt and favicon changes rely on users being able to see the icons, especially since these changes are not surfaced to the user unless their focus happens to already be on those icons when they change. This makes knowing the status of a cell difficult to impossible for users who cannot look at the screen in full. Quick to run code cells with a cell output—like the majority of cells run during these tests— tend to make this a trivial issue, it marks a drastic experience difference for sighted and non-sighted users. Fully sighted users can quickly verify the status of a cell in two different places, go about other work while waiting, and more easily verify when a cell has completed running. 
```

Comment:
Good to know! Link added.

```suggestion
**Current default JupyterLab feedback for notebook cell status is exclusively visual**; the cell prompt and [favicon](https://en.wikipedia.org/wiki/Favicon) changes rely on users being able to see the icons, especially since these changes are not surfaced to the user unless their focus happens to already be on those icons when they change. This makes knowing the status of a cell difficult to impossible for users who cannot look at the screen in full. Quick to run code cells with a cell output—like the majority of cells run during these tests— tend to make this a trivial issue, it marks a drastic experience difference for sighted and non-sighted users. Fully sighted users can quickly verify the status of a cell in two different places, go about other work while waiting, and more easily verify when a cell has completed running. 
```

Reply:
> Is it a common web app pattern to reflect app status in the favicon?

I don't actually know. Off the top of my head I can't think of others that do it other than adding notifications symbols over the normal favicon.

> I would link to the JupyterLab docs, but I don't see this behavior documented there (except in the changelog)

Good to know it wasn't just me then! 😆 

```suggestion
**Current default JupyterLab feedback for notebook cell status is exclusively visual**; the cell prompt and [favicon changes](https://github.com/jupyterlab/jupyterlab/pull/4361) rely on users being able to see the icons, especially since these changes are not surfaced to the user unless their focus happens to already be on those icons when they change. This makes knowing the status of a cell difficult to impossible for users who cannot look at the screen in full. Quick to run code cells with a cell output—like the majority of cells run during these tests— tend to make this a trivial issue, it marks a drastic experience difference for sighted and non-sighted users. Fully sighted users can quickly verify the status of a cell in two different places, go about other work while waiting, and more easily verify when a cell has completed running. 
```

> Or maybe the kernel status reflected in the favicon is already common knowledge, and I'm just late to the party. 

Not all participants were aware of it, so I think it's good to clarify. My personal guess would be that it's not known by everyone. I'm glad you called it out.

Thank you for finding that link!

Reply:
ha! I have been using notebooks/lab for years and I **never** noticed the change to an hourglass 🫠 I suppose its placement is not an obvious place to search for "status" cues